public/app/modules/home/home.controller.js

```diff
@@ -50,16 +50,22 @@
             }, function(error){
                     $scope.page_content = {};
                 });
+
             /**
-             * Retrieve all resources (pages and posts) which are flagged
-             * for display within the carousel.
+             * Retrieve the carousel based on name
              */
-            dataService.getCarouselItems()
-                .query({carousel: 'yes', sortResultBy: 'positionInCarousel'}, function(carousel_items){
-                $scope.slides = carousel_items;
-            }, function(error){
-                $scope.slides = {};
-            });
+            dataService.getCarousel()
+                .show({name:'HomePage'}, function(carousel) {
```

Comment:
@matjames007 to avoid hard coding the name of the carousel, I think we would set a flag on the carousel model to indicate whether its published or not. That way the query doesn't depend on a name.